### ISSUE TITLE
Close button position

### ISSUE THREAD

**Issue by viliusle** [rqrauhvmra/Tobii#85]
In most cases close button in top-right corner works.  But not in all cases.  If we have lots of small images or using inline HTML feature (modal window will be small) or having big screen, close button should be closer to actual content. User focus will be in center, and to exit, he will need to waste time searching for close button and move mouse to close button.

The problem, it is hard to achieve it even with custom CSS, because current HTML layout will not let you do this easily. We should try to find a way to solve it. 

p.s. I am not suggest to change it globally, but to have such ability with `data-` attribute. It would probably be breaking change.

**Comment by midzer** [comment]
Do you think about having something like `data-position="top-left"` for each element to set the close button position? This would mean we can override current close button css via JavaScript. What positions do you want to offer?

If we leave `top-right` as default, there would be no breaking change, as it is just optional.

**Comment by viliusle** [comment]
Possible values would be only 2, I do not see why we should add more options.

- absolute top-left (default, current)
- relative top-left (right above image top right corner)

**Comment by midzer** [comment]
Yeah, if you need this feature @viliusle just open up a PR and we can discuss the changes.

**Comment by viliusle** [comment]
I do not need it anymore, because I found a way using hacky CSS, but problem remains. Still it is hard to fix it in current version, so low priority I guess.
